tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -536,7 +538,8 @@ def do_rolling_bounce(self, processor, counter, current_generation):
                     self.old_processors.remove(processor)
                     self.upgraded_processors.append(processor)
 
-                    log_monitor.wait_until("Kafka version: " + str(DEV_VERSION),
+                    # checking for the dev version which should be the only SNAPSHOT
+                    log_monitor.wait_until("Kafka version.*-SNAPSHOT",
                                            timeout_sec=60,
                                            err_msg="Could not detect Kafka Streams version " + str(DEV_VERSION) + " in " + str(node.account))
```

Comment:
Shouldn't `str(DEV_VERSION)` in the error message also be changed?

Reply:
I wanted to leave the `str(DEV_VERSION)` as is so it's more clear in the logs if an error occurs